docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+   EOF  
+   ```
+
+- Create a DNSEntry CR. See the following example and modify values for the **metadata.namespace**, **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSEntry
+   metadata:
+     name: dns-entry
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     dnsName: "my.sub1.mydomain.com" # Your subdomain. Edit this value
+     ttl: 600
+     targets:
+       - 1.2.3.4 # IP address. Edit this value
+   EOF
+   ```
+
+    >**NOTE:** To check the Ingress Gateway IP, run:
+    >```
+    >kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    >```
+
+   >**NOTE:** You can create many DNSEntry CRs for one DNSProvider, depending on the number of subdomains you want to use.
+
+5. Create an Issuer CR. See the following example and modify values of the **metadata.spec.acme.email**, **metadata.spec.domains.include**, and **metadata.spec.domains.exclude** parameters. As the values for the **metadata.spec.domains.include** parameter, use the main domain, the subdomain from the **spec.dnsName** parameter of the DNSEntry CR, and a wildcard DNS record. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: cert.gardener.cloud/v1alpha1
+   kind: Issuer
+   metadata:
+     name: letsencrypt-staging
+     namespace: default
+   spec:
+     acme:
+       server: https://acme-staging-v02.api.letsencrypt.org/directory
+       email: YOUR_EMAIL_ADDRESS # Edit this value
+       autoRegistration: true
+       # Name of the Secret used to store the ACME account private key
+       # If doesn't exist, a new one is created
+       privateKeySecretRef:
+         name: letsencrypt-staging-secret
+         namespace: default
+       domains:
+         include:
+           - mydomain.com # The main domain. Edit this value
+           - my.sub1.mydomain.com # The subdomain provided in the DNS Entry created in the previous step. Edit this value
+           - "*.mydomain.com" # A wildcard DNS record. Edit this value
+         # Optionally, restrict domain ranges for which certificates can be requested
+   #     exclude:
+   #       - my.sub2.mydomain.com # Edit this value
+   EOF
+   ```
+
+6. Create a Certificate CR. See the following example and modify values of the **spec.secretName**, **spec.commonName**, and **spec.issuerRef.name**, and **spec.dnsNames** parameters. As the value for the **spec.issuerRef.name** parameter, use the value from te **metadata.name** parameter of the Issuer CR. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: cert.gardener.cloud/v1alpha1
+   kind: Certificate
+   metadata:
+     name: httpbin-cert
+     namespace: istio-system
```

Comment:
Shouldn't it be deployed in the same namespace as user provided?
Otherwise following docs to expose the service, you will receiver the error that gateway cant find the credentials.

Reply:
The Certificate is supposed to create a Secret that has to land in `Istio-system` to let Istio recognise it. Beforehands `kubed` was used to sync the secret between the user's namespace and the `Istio-system`. Now it is not needed anymore and we can create it there directly.

Reply:
TODO: We need to verify this.